enhancements/installer/aws-custom-edge-machineset-local-zones.md

```diff
@@ -0,0 +1,665 @@
+---
+title: aws-custom-edge-machineset-local-zones
```

Comment:
@patrickdillon @JoelSpeed @makentenza @rvanderp3 could you please take a look at this EP?

Reply:
Also @dhellmann, Thanks!

Reply:
@elmiko would be happy with your thoughts here too. Thanks! =]

Reply:
@mtulio sorry, just saw this ping. will take a read